src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -17,9 +17,13 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "doctrine/common": "~2.4",
         "symfony/polyfill-ctype": "~1.8",
-        "symfony/polyfill-mbstring": "~1.0"
+        "symfony/polyfill-mbstring": "~1.0",
+        "doctrine/annotations": "~1.0",
+        "doctrine/cache": "~1.6",
+        "doctrine/collections": "~1.0",
+        "doctrine/event-manager": "~1.0",
+        "doctrine/persistence": "~1.0"
```

Comment:
I think the only direct dependency of the doctrine bridge in doctrine/common is actually `doctrine/persistence`

Reply:
and this means the root composer.json would depend on persistence, cache and annotations.